src/mbed/main.cpp

```diff
@@ -321,6 +328,12 @@ bool sendResponse(TCPSocket &client)
   response.left_output = g_left_output;
   response.right_output = g_right_output;
 
+  response.desired_speed_l = g_desired_speed_l;
+  response.desired_speed_r = g_desired_speed_r;
+
+  response.debug.arg = (void*)""; // set debug message here
+  response.debug.funcs.encode = &encode_string;
```

Comment:
Having to do variable length stuff seems a bit annoying.... Maybe have some fixed length char array so its easy encode and decode?